tests/acceptance/features/apiCollaboration/wopi.feature

```diff
@@ -522,3 +522,33 @@ Feature: collaboration (wopi)
       | app-endpoint                                              |
       | /app/open-with-web?file_id=<<FILEID>>&app_name=FakeOffice |
       | /app/open-with-web?file_id=<<FILEID>>                     |
+
+
+  Scenario Outline: open file with non-existing file id (open-with-web)
+    Given user "Alice" has uploaded file "filesForUpload/simple.odt" to "simple.odt"
+    And we save it into "FILEID"
+    And user "Alice" has deleted file "/simple.odt"
+    When user "Alice" sends HTTP method "POST" to URL "<app-endpoint>"
```

Comment:
```suggestion
    When user "Alice" tries to send HTTP method "POST" to URL "<app-endpoint>"
```

negative scenario should have **tries to** step. In past, we converted every negative test scenario `when` step into `tries to` so could you plz do that here too and other scenario of this file

Reply:
I understand that in the past we've used "tries to" for negative scenarios to emphasize the action. However, in this case, I felt that "tries to" was redundant since the failure is inherent in the scenario (i.e., opening a file with a non-existing file ID guarantees failure).

That being said, I'm happy to align with the team's convention for consistency. I'll update the scenario as suggested. However, I wanted to clarify my thought process, which was that "tries to" is more useful when there's a possibility of success under different conditions (e.g., valid credentials). In this case, failure is guaranteed by the invalid input.

Let me know if this reasoning makes sense, and I'll proceed with the update to keep things consistent across the project!

Reply:
>"tries to" is more useful when there's a possibility of success under different conditions

We use `tries to` for the actions which fail. For example: if a non-admin user is trying to create new user, we would write `user tries to create...`.

But for this particular step, I don't have any preferences because it is indicating to make a request. So if we use `sends` or `tries to send` both seem ok. since the request is successfully sent, it can justify `sends` and since the response we get is error response, it can justify `tries to`.
If the step was `When user opens the file...` then we had to use `user tries to open ...`.

Reply:
so it's clear we need to add **tries to** for the failing scenario. I hope everyone remembers that for now on 😄. Plz do change if their is similar case in this file